app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -41,6 +41,7 @@ define(function (require, exports, module) {
       this.closeAllPanels();
 
       this.$('.settings-unit').addClass('open');
+      this.parentView.$('#fxa-settings-content').addClass('animate-shadow');
```

Comment:
Reaching into the parentView to fetch a child on which to operate seems like a round about way of doing things... A subtle change that makes fewer assumptions about this parent is to call [this.$el.closest('#fxa-settings-content')](https://api.jquery.com/closest/).